flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/ShowCreateUtil.java

```diff
@@ -266,7 +277,7 @@ private static String formatComment(String comment) {
     }
 
     private static String formatPartitionedBy(String partitionedByColumns) {
-        return String.format("PARTITION BY (%s)\n", partitionedByColumns);
+        return String.format("PARTITIONED BY (%s)\n", partitionedByColumns);
```

Comment:
should be partitioned, since it was used only for materialized tables from recent commit, should be ok to fix